subsys/bluetooth/mesh/adv.h

```diff
@@ -71,9 +72,10 @@ int bt_mesh_scan_disable(void);
 int bt_mesh_adv_enable(void);
 
 void bt_mesh_adv_buf_local_ready(void);
-
```

Comment:
Any reason why you're removing this line, and thereby grouping `local` and `relay` together, but not `friend`?

Reply:
No, just an oversight by me. Will fix 👍